mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/print/config.yaml

```diff
@@ -37,7 +37,7 @@ scales:
   - 559082264
 
 # see https://github.com/georchestra/georchestra/issues/362
-brokenUrlPlaceholder: http://sdi.georchestra.org/public/transparent.gif
+brokenUrlPlaceholder: https://sdi.georchestra.org/public/transparent.gif

```

Comment:
> Good point.
> Maybe we should document how to import their certificate into the JVM truststore ?

```
openssl s_client -connect site.fqdn:443 </dev/null 2>/dev/null|openssl x509 -out /tmp/site.fqdn.crt
keytool -importcert -alias site.fqdn -file /tmp/site.fqdn.crt -keystore /path/to/keystore -storepass keystore.pass -noprompt

```
The first line is not needed if you already have the crt, but i used this to import remote certs for geonetwork harvesting over https